### PR TITLE
Feature/cli update tip

### PR DESCRIPTION
**Pull request by nikewu**
1. 检测到有新版本时，进行版本更新提示，如果选择是则如图所示进行更新安装
企业微信截图_15746892368754
2. 如果选择否则一天内不再提示

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/feflow?pullRequest=153) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/feflow?pullRequest=153) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: nikewu<br/>:x: yanzhanghu<br/><hr/>**yanzhanghu** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/feflow?pullRequest=153) it.</sub>

File: packages/feflow-cli/src/cli/upgrade.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import semver from 'semver';
+import { safeDump } from '../shared/yaml';
+import spawn from 'cross-spawn';
+import inquirer from 'inquirer';
+
+async function updateCli(packageManager: string) {
+  return new Promise((resolve, reject) => {
+    const args = [
+      'install',
+      'feflow-cli@latest',
```
Comment (cpselvis) [inline packages/feflow-cli/src/cli/upgrade.ts]: 修改成 @feflow/cli

File: packages/feflow-cli/src/cli/upgrade.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    return;
+  }
+  const packageManager = config.packageManager;
+  const registryUrl = await getRegistryUrl(packageManager);
+  const latestVersion: any = await packageJson('feflow-cli', 'latest', registryUrl)
+  if (semver.gt(latestVersion, version)) {
+    const askIfUpdateCli = [{
+      type: "confirm",
+      name: "ifUpdate",
+      message: `${chalk.yellow(`feflow-cli's latest version is ${chalk.green(`${latestVersion}`)} but your version is ${chalk.red(`${version}`)}, Do your want to update it?`)}`,
```
Comment (cpselvis) [inline packages/feflow-cli/src/cli/upgrade.ts]: 修改成 @feflow/cli

File: packages/feflow-cli/src/cli/upgrade.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+export default async function checkCliUpdate(ctx: any) {
+  const { version, config, configPath } = ctx;
+  if (config.lastUpdateCheck && (+new Date() - parseInt(config.lastUpdateCheck, 10)) <= 1000 * 3600 * 24) {
+    return;
+  }
+  const packageManager = config.packageManager;
+  const registryUrl = await getRegistryUrl(packageManager);
+  const latestVersion: any = await packageJson('feflow-cli', 'latest', registryUrl)
```
Comment (cpselvis) [inline packages/feflow-cli/src/cli/upgrade.ts]: 修改成 @feflow/cli

File: packages/feflow-cli/src/core/index.ts [hunk 4]
```diff
@@ -261,4 +265,59 @@ export default class Feflow {
...
+            return;
+        }
+        const packageManager = config.packageManager;
+        const registryUrl = await getRegistryUrl(packageManager);
+        const latestVersion: any = await packageJson('feflow-cli', 'latest', registryUrl)
+        if (semver.gt(latestVersion, version)) {
+            const askIfUpdateCli = [{
+                type: "confirm",
+                name: "ifUpdate",
+                message: `${chalk.yellow(`feflow-cli's latest version is ${chalk.green(`${latestVersion}`)} but your version is ${chalk.red(`${version}`)}, Do your want to update it?`)}`,
```
Comment (cpselvis) [inline packages/feflow-cli/src/core/index.ts]: 改成 @feflow/cli

Comment (nikewu) [reply packages/feflow-cli/src/core/index.ts]: 好

**Pull request merged**